### PR TITLE
Fixes #21003 - Improve table for ansible roles

### PR DESCRIPTION
**Pull request by bastilian**
Before:
gnome-shell-screenshot-7sit9y

After:
gnome-shell-screenshot-p3za9y

### CODE REVIEW

File: app/controllers/ansible_roles_controller.rb [hunk 1]
```diff
@@ -5,10 +5,11 @@ class AnsibleRolesController < ::ApplicationController
   before_action :find_resource, :only => [:destroy]
   before_action :find_proxy, :only => [:import]
   before_action :create_importer, :only => [:import, :confirm_import]
+  before_action :default_order, :only => [:index]
 
   def index
     @ansible_roles = resource_base.search_for(params[:search],
-                                              :order => params[:order]).
+                                              :order => ()).
```
Comment (dLobatog) [inline app/controllers/ansible_roles_controller.rb]: Shouldn't this still be `params[:order]` in order to use `default_order`?

**Comment by ares** [comment]
Could we also have columns containing number/date more thin? values there have fixed size, the name though can be quite long. Could you please add ordering to the rest of columns? At lesat imported_at if counts are too complicated.

File: app/controllers/ansible_roles_controller.rb (hunk 1 above)
Comment (bastilian) [reply app/controllers/ansible_roles_controller.rb]: Yes, it should. fixed.

**Comment by bastilian** [comment]
@ares & @dLobatog:

* Adjusted the width of all colums
* Ordering by "Imported at" is now possible
* Added scoped_search for ansible_roles to `Host` to allow searching for `role =`
* Hosts count is linked to roles search of hosts

gnome-shell-screenshot-7mzo9y

**Comment by dLobatog** [comment]
@bastilian I think https://github.com/theforeman/foreman_ansible/commit/62c71aa1ecbff4757584d8800d75b34ec0f4e943 was causing one test to fail. If so, we should see http://ci.theforeman.org/job/test_plugin_foreman_ansible_master/ green again (at least I do locally). If it was something else, I'll merge this & work on the error after.

**Comment by dLobatog** [comment]
Thanks @bastilian !

**Pull request merged**